widget/accordioncontainer_test.go

```diff
@@ -0,0 +1,235 @@
+package widget
+
+import (
+	"testing"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	style      = fyne.TextStyle{}
+	foo1Min    = fyne.MeasureText("foo1", theme.TextSize(), style)
+	foo2Min    = fyne.MeasureText("foo2", theme.TextSize(), style)
+	foo3Min    = fyne.MeasureText("foo3", theme.TextSize(), style)
+	foobar1Min = fyne.MeasureText("foobar1", theme.TextSize(), style)
+	foobar2Min = fyne.MeasureText("foobar2", theme.TextSize(), style)
+	foobar3Min = fyne.MeasureText("foobar3", theme.TextSize(), style)
+)
+
+func addItem(t *testing.T, ac *AccordionContainer, header, detail string) {
+	t.Helper()
+	ac.Append(header, &canvas.Text{Text: detail, TextSize: theme.TextSize()})
+}
+
+func TestAccordionContainer(t *testing.T) {
+	t.Run("Initializer", func(t *testing.T) {
+		ac := &AccordionContainer{}
```

Comment:
To test initializer it really should pass the items in as well.
Imagine that this is being loaded from a JSON file or other serialised data - can the object be created without calling any functions for setup?

Reply:
Done